public/class-pw-admin-public.php

```
@@ -0,0 +1,1606 @@
+            }
+        }
+        return $item_data;
+    }
+
+    /**
+     * Display the custom image directly in the cart item row, after the name.
+     *
+     * @param array $cart_item Cart item data.
+     * @param string $cart_item_key Cart item key.
+     */
+    public function add_custom_text_after_cart_item_name($cart_item, $cart_item_key)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$cart_item_key'. (Unused Code Rules)

(UnusedFormalParameter)